### ISSUE TITLE
[4.x]: updating field value via resave/entries on console doesn't invalidate element index cache

### ISSUE THREAD

**Issue by vnali** [craftcms/cms#11050]
### What happened?

it seems updating field value via resave/entries on console doesn't invalidate element index cache and i can't see new values after refreshing index page.
i use this to update field: `php craft resave/entries --set exampleField --to "=new value" `

i checked Craft 3 and it seems element index cache show changes immediately after resave and then refreshing page.

### Craft CMS version

4.0.0-RC2

### Installed plugins and versions

-

**Comment by brandonkelly** [comment]
I’m not able to reproduce this. The index cache is getting cleared as expected via `resave/entries`, but not when I change the values directly in the database.

Is the cache updating as expected when you save entries normally?

**Comment by vnali** [comment]
@brandonkelly yes, on saving one element via edit page, all custom field new values appear on element index.

**Comment by vnali** [comment]
@brandonkelly uploaded a video, tested with fresh Craft 4 RC2 install

https://user-images.githubusercontent.com/55586085/166224228-f172f02b-4bb6-4d1c-8bf1-05ecfab21963.mp4

**Comment by brandonkelly** [comment]
I’m still not able to reproduce this, and have stepped through the code to double-check that everything is actually working as expected, to be sure it’s not a fluke. The results are definitely loading from cache when nothing has changed, and the cache is definitely getting cleared when the element is updated via the CLI.

I’ve also asked others internally to try to reproduce, but it’s working as expected for them too.

I’m guessing it’s a file permissions issue. Perhaps your CLI user is different from the web user, and the CLI user doesn’t have permission to write changes to your `storage/runtime/cache/` directory.

**Comment by vnali** [comment]
weirdly i am not able to reproduce it now. yeah, maybe it was permission issue. thank you for you time @brandonkelly
